server/api/controllers/users/update-user.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  friendlyName: 'View users',
+
+  description: '',
+
+  inputs: {
+    params: {
+      required: false,
+      type: 'json'
+    }
+  },
+
+  exits: {
+    success: {
+      description: "User account returned successfully",
+    },
+    invalid : {
+      description: "Failed to retrieve user account",
+    }
+  },
+  fn: async function({ userId }, exits) {
+    User.updateOne(params.id, params)
```

Comment:
i'm guessing this file wasn't supposed to be in this PR, if so, can you remove it from this PR?